src/Assets/styleController.ts

```diff
@@ -17,6 +18,7 @@ export interface ITheme {
     'font-size': string;
     '--main-header-color': string;
     '--midBlue-color': string;
+    '--brightOrange-color': string;
```

Comment:
I called this `--icon-color` on my `Step-5` branch. Feel free to reach out if you have any questions.